config.toml

```diff
@@ -926,12 +926,14 @@
     MinPeersThreshold                                = 0.8  # 80%
     DelayBetweenRequestsInSec                        = 5   # 5sec
     MaxTimeoutInSec                                  = 7200 # 2h
-    DelayBetweenConnectionNotificationsInSec         = 60   # 1min
+    PeerShardTimeBetweenSendsInSec                   = 1800   # 0.5h
```

Comment:
We should check if this value is correct.

Reply:
yes, we can live with this set to 30minutes